src/server/main.js

```diff
@@ -37,7 +37,7 @@ io.on('connect', (socket) => {
   // check if token exists
   let socketUser = false;
   if (socket.handshake.query.token) {
-    socketUser = authentication.verifyToken(socket.handshake.query.token);
+    socketUser = authentication.verifyToken(socket.handshake.query.token).value;
```

Comment:
Make sure you check `isSuccesful` here. Client might have sent a bad token. As is, value would be a string error message but we want `socketUser` to still be false if the token is bad.